### ISSUE TITLE
Zero bot is very slow after refusing the ticket.

### ISSUE THREAD

**Issue by Guseyn** [zerocracy/farm#2243]
There is an example of very slow reaction from zero-bot:
https://github.com/artipie/npm-adapter/pull/33

I created pr, after **6** hours zero bot assigns the reviewer. As you can see, @fanifieiev refused reviewing the pr, after **5** hours zero-bot reacts on this situation, after another **10** hours zero-bot writes another comment about refusing. And it does not assign another free reviewer. It's still not assigned to anyone.

In total, we have lost **21!!** hours(by the moment of this issue is filed) just only because of slow reactions from zero-bot.

Obviously, zero bot has major problem that cannot be ignored. I think it's unacceptable to have such slow process in development. It must be fixed and speed of reaction must be increased.

Plz @yegor256 @g4s8 could you clarify exactly why it's so slow, and how it can be fixed.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@guseyn[/z](https://www.0crat.com/u/guseyn) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by Guseyn** [comment]
Updated information for the issue:

After the review refused the reviewing the pr, zero-bot assigned the ticket only after **2** days.

```
fanifieiev commented 3 days ago
@0crat refuse
```

```
0crat commented yesterday
This pull request #33 is assigned to @dgarus/z, ...
```

**Comment by g4s8** [comment]
@Guseyn thanks for reporting, it's a known issue, we're going to fix it soon. Closing as a duplicate of #2240

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#2243` is not assigned, can't get performer

**Comment by Guseyn** [comment]
@g4s8 why do we need zero-bot then of it does not work properly, I think it must be fixed and only after it can be used.
